src/lib/phrases/infrastructure/components/phrase.svelte

```diff
@@ -5,9 +5,11 @@
   import relativeTime from "dayjs/plugin/relativeTime";
   dayjs.extend(relativeTime);
 
+  export let id: bigint;
   export let content = "";
   export let lastUpdate: Date;
   export let removeable = false;
+  export let onClose: (id: bigint) => void;
```

Comment:
**issue:** follow svelte events dispatch instead

Svelte handles events in a different way from React. Instead of passing handlers, the component dispatches its events by using [dispatchers](https://svelte.dev/docs#template-syntax-component-directives-on-eventname). For more information read following link:
https://svelte.dev/tutorial/component-events